pages/organizer/[name].tsx

```diff
@@ -46,30 +46,38 @@ const OrganizerPeople: FC<Volunteer> = ({
         <p className="text-muted fs-6">{motto}</p>
       </Col>
     </Row>
-    <h2>我与FCC的故事</h2>
-    <Row className="mt-4 fs-5 lh-base">
-      {storyWithFCC.map(item => (
-        <Col className="mt-4" key={item}>
-          {item}
-        </Col>
-      ))}
-    </Row>
+    <section>
+      <h2 className="py-3">我与FCC的故事</h2>
+      <Row as="ul" className="mt-1 fs-5 list-unstyled" xs={1}>
+        {storyWithFCC.map(item => (
+          <Col as="li" key={item}>
+            <p className="first-line-indent">{item}</p>
+          </Col>
+        ))}
+      </Row>
```

Comment:
```suggestion
        {storyWithFCC.map(item => (
            <p key={item} className="first-line-indent fs-5 mt-1">{item}</p>
        ))}
```
纵向一列用布局框没意义，多个 `<p />` 自然排下来即可。